manifests/worker.pp

```diff
@@ -271,8 +272,20 @@
     }
   }
 
+  # Add compatibility redirects
+  if $name =~ /^(centos)\-(.*)/ and $server_type == "passive"  {
```

Comment:
We should use single quotes for consistency, any time there is no variable substitution inside the string.